R/pkg/tests/fulltests/test_sparkSQL.R

```diff
@@ -1762,6 +1762,28 @@ test_that("column functions", {
     collect(select(df, alias(not(df$is_true), "is_false"))),
     data.frame(is_false = c(FALSE, TRUE, NA))
   )
+
+  # Test percentile_approx
+  actual <- lapply(
+    list(
+      percentile_approx(column("foo"), 0.5),
+      percentile_approx(column("bar"), 0.25, 42L),
+      percentile_approx(column("bar"), c(0.25, 0.5, 0.75)),
+      percentile_approx(column("foo"), c(0.05, 0.95), 100L),
+      percentile_approx("foo", 0.5),
+      percentile_approx("bar", c(0.1, 0.9), 10L)),
+    function(x) SparkR:::callJMethod(x@jc, "toString"))
+
+  expected <- list(
+     "percentile_approx(foo, 0.5, 10000)",
+     "percentile_approx(bar, 0.25, 42)",
+     "percentile_approx(bar, [0.25,0.5,0.75], 10000)",
+     "percentile_approx(foo, [0.05,0.95], 100)",
+     "percentile_approx(foo, 0.5, 10000)",
+     "percentile_approx(bar, [0.1,0.9], 10)"
+  )
+
+  exepct_equal(actual, expected)
```

Comment:
This should be sufficient, as the main concern is if data is properly passed to JVM.